package.json

```diff
@@ -3,16 +3,10 @@
   "version": "0.1.0",
   "description": "UMA Protocol Smart Contracts",
   "scripts": {
-    "lint": "npm run eslint && npm run prettier",
-    "lint_check": "npm run eslint_check && npm run prettier_check",
-    "lint_check_quick": "npm run eslint_check && npm run prettier_check_quick",
-    "eslint": "npm run eslint_vanilla -- --fix",
-    "eslint_check": "npm run eslint_vanilla",
-    "eslint_vanilla": "eslint 'core/**/*.js'",
-    "prettier": "COMMAND=prettier npm run prettier_vanilla -- --write",
-    "prettier_check": "COMMAND=prettier npm run prettier_vanilla -- --check",
-    "prettier_check_quick": "COMMAND=pretty-quick npm run prettier_vanilla -- --check",
-    "prettier_vanilla": "$COMMAND --bracket-spacing 'core/**/*.js' 'common/**/*.js' '*.js' 'voter-dapp/src/**/*.js' 'sponsor-dapp-v2/src/**/*.js' 'core/contracts/**/*.sol' 'financial-templates-lib/*.js' 'liquidator/**/*.js' 'disputer/**/*.js'"
+    "lint": "npm run eslint && npm run prettier -- --list-different",
```

Comment:
This is much cleaner now. you can only `lint` or `lint-fix`. Both commands use eslint and prettier under the hood for their respective roles (see main PR comment).

Reply:
Just to triple check -- this command outputs a nonzero exit code if it finds issues, right?

Reply:
Ye. this throws basically the same way that it did before. I updated the ci bash script as well.